static/src/javascripts/projects/common/modules/commercial/ads/user-ad-feedback.js

```diff
@@ -3,24 +3,33 @@ define([
     'qwery',
     'fastdom',
     'common/utils/fetch',
-    'common/utils/config'
+    'common/utils/config',
+    'common/utils/detect'
 ], function (
     bonzo,
     qwery,
     fastdom,
     fetch,
-    config
+    config,
+    detect
 ) {
 
-    return function recordUserAdFeedback(pagePath, adSlotId, creativeId, feedbackType) {
+    return function recordUserAdFeedback(pagePath, adSlotId, slotRenderEvent, feedbackType) {
         var feedbackUrl = 'https://j2cy9stt59.execute-api.eu-west-1.amazonaws.com/prod/adFeedback';
         var stage = config.page.isProd ? 'PROD' : 'CODE';
+        var ua = detect.getUserAgent;
+        var breakPoint = detect.getBreakpoint();
+
         var data = JSON.stringify({
             stage: stage,
             adPage: pagePath,
             adSlotId: adSlotId,
-            adCreativeId: creativeId.toString(),
-            adFeedback: feedbackType
+            adCreativeId: slotRenderEvent.creativeId.toString(),
+            adLineId: slotRenderEvent.lineItemId.toString(),
+            adFeedback: feedbackType,
+            browser: ua.browser.toString() + ua.version.toString(),
```

Comment:
Fair enough!

On Mon, 15 Aug 2016 at 17:00 Justin Pinner <notifications@github.com> wrote:

> In
> static/src/javascripts/projects/common/modules/commercial/ads/user-ad-feedback.js
> <https://github.com/guardian/frontend/pull/13996#discussion_r74784783>:
>
> >          var data = JSON.stringify({
> >              stage: stage,
> >              adPage: pagePath,
> >              adSlotId: adSlotId,
> > -            adCreativeId: creativeId.toString(),
> > -            adFeedback: feedbackType
> > +            adCreativeId: slotRenderEvent.creativeId.toString(),
> > +            adLineId: slotRenderEvent.lineItemId.toString(),
> > +            adFeedback: feedbackType,
> > +            browser: ua.browser.toString() + ua.version.toString(),
>
> Thought it might be useful in case there are any discrepancies between the
> UA string and how we interpreted it. It's also quick to read 👓
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/guardian/frontend/pull/13996/files/a6617ce32ee8c623d6ec90e368aba7a1705b66b8#r74784783>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAmc2EfA-PcLcsJC7fqoDVlHtFabmk7Tks5qgI0ygaJpZM4Jkfm_>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396